packages/calendar-input/src/date-input.tsx

```diff
@@ -289,6 +290,7 @@ export const DateInput: React.FC<DateInputProps> = ({
       onClick={handleClick}
       appearance={props.appearance}
       onKeyDown={handleKeyDown}
+      disabled={disabled}
```

Comment:
If it is a boolean can't this be passed as just a `disabled` props?

Reply:
nope, here I'm passing a value - without it it's always `true`